vsdkx/core/detector.py

```diff
@@ -38,6 +39,9 @@ def __init__(self,
                                       "model.settings")
         model_profile = get_env_dict(system_config,
                                      "model.profile")
+        self.image_type = get_env_dict(system_config,
+                                       "image_type",
+                                       default=None)
```

Comment:
A suggestion is to set the default here to 'BGR' instead of None and you if could have only one condition later in the code.

It's just a suggestion though. you can ignore it if you didn't feel like to change it.

Reply:
yeah I agree with that

Reply:
Yeah, good suggestion. I will change the code accordingly